akka-http-core/src/main/scala/akka/http/javadsl/model/MediaType.scala

```diff
@@ -89,6 +89,8 @@ trait MediaType {
 
   def isVideo: Boolean
 
+  def isFont: Boolean
```

Comment:
Hmm, indeed this produces a binary incompatibility because we did not mark `MediaType` as `DoNotInherit` (though I think it was meant that way).

The easiest way around this would be not to introduce `isFont` - it is not directly needed anywhere, is it?

Reply:
Not needed directly. I think it will be better if we add that as soon as possible, because it causing inconsistance in code